signac/contrib/utility.py

```diff
@@ -188,6 +267,29 @@ def _mkdir_p(path):
 
 
 def split_and_print_progress(iterable, num_chunks=10, write=None, desc='Progress: '):
+    """Split the progress and prints it.
```

Comment:
what is the usage of this function?

Reply:
The only place I see this called is when `Project.upate_cache` is called. This seems to be for debugging purposes and gives the progress/time taken to cache Project information.

Reply:
thanks for the explanation!